server/sonar-server/src/main/java/org/sonar/server/debt/DebtModelPluginRepository.java

```diff
@@ -28,6 +28,7 @@
 import java.util.Collections;
 import java.util.Map;
 import org.picocontainer.Startable;
+import org.sonar.api.Plugin;
 import org.sonar.api.SonarPlugin;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.api.SonarPlugin'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)